infra/grafana/dashboards/jvm-micrometer.json

```
@@ -0,0 +1,363 @@
+          "legendFormat": "{{method}} {{uri}} {{status}}",
+          "refId": "A"
+        }
+      ],
+      "title": "HTTP Request Rate (req/s)",
+      "type": "timeseries"
+    },
+    {
+      "datasource": { "type": "prometheus", "uid": "${DS_PROMETHEUS}" },
+      "fieldConfig": {
+        "defaults": {
+          "color": { "mode": "palette-classic" },
```

Biome (2.4.15)

[error] The key color was already declared.

(lint/suspicious/noDuplicateObjectKeys)